showcase.html

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+desc: A showcase of websites and applications using Vanilla Framework
+sitemap:
+    priority: 1.0
+    changefreq: 'monthly'
+    lastmod: 2018-11-20T10:27:30+00:00
+---
+
+<section id="main-content" class="p-strip--image is-dark" style="background-image: url('https://assets.ubuntu.com/v1/775cc62b-vanilla-grad-background.png'); background-position: 75% 50%;">
```

Comment:
@deadlight could you consider changing the inline style to be consistent with your page design. I noticed that inline styles are not used anywhere else in the code. Might be better using style tags  and then adding a class name to the section.